helm-charts/common/llm-vllm-uservice/README.md

```diff
@@ -0,0 +1,87 @@
+# llm-vllm Microservice
+
+Helm chart for deploying a microservice which facilitates connections and handles responses from OpenVINO vLLM microservice.
+
+`llm-vllm-uservice` depends on OpenVINO vLLM. You should properly set `vLLM_ENDPOINT` as the HOST URI of vLLM microservice. If not set, it will consider the default value : `http://<helm-release-name>-vllm-openvino:80`
```

Comment:
I remember hit a limitation of 15 characters for the Chart name, maybe this is no longer a limitation.

Reply:
Was able to work with longer names, so doesn't seem like a limitation any more.